pom.xml

```diff
@@ -45,62 +42,51 @@
             </snapshots>
             <id>mdht-libs</id>
             <name>libs-release</name>
-			<!-- <url>https://artifactory-corp.cerner.com/artifactory/ais-repo</url> -->
 			<url>http://artifactory-corp.cerner.com/artifactory/ais-repo</url>
+			<!-- <url>https://artifactory-corp.cerner.com/artifactory/ais-repo</url> -->
         </repository>
-        <!--
-		<repository>
-            <snapshots>
-                <enabled>true</enabled>
-            </snapshots>        
-            <id>central</id>
-            <name>ONCSITE-DEV-CCDA-releases</name>
-            <url>http://ccda.dev.sitenv.org:8081/artifactory/libs-release-local</url>
-        </repository>
-		-->
     </repositories>
-
     <dependencies>
         <!-- START MDHT DEPENDENCIES - from devsoap.sitenv.org:8081 artifactory -->
 		<!-- 1st party MDHT - MODELS from https://github.com/mdht/mdht-models -->
  		<dependency>
 		    <groupId>org.openhealthtools.mdht.cda</groupId>
 		    <artifactId>org.openhealthtools.mdht.uml.cda.consol2</artifactId>
-		    <version>2.6.3.20170323</version>		    
+		    <!-- <version>2.6.3.20170323</version> -->
+		    <version>3.0.9.20180622</version>
 		</dependency> 
 		<dependency>
 		    <groupId>org.openhealthtools.mdht.cda</groupId>
 		    <artifactId>org.openhealthtools.mdht.uml.cda.mu2consol</artifactId>
-		   <version>2.6.3.20170323</version>		   		    
+		   <!-- <version>2.6.3.20170323</version> -->
+		   <version>3.0.8.20180531</version>	
 		</dependency>
 		

```

Comment:
I still think we should add below 2 jars that's from recent MDHT releases and ETT Validator releases:

        <dependency>
            <groupId>mdht.hl7.ds4p</groupId>
            <artifactId>org.hl7.cbcc.privacy.consentdirective</artifactId>
            <version>${consentdirective.version}</version>
        </dependency>
        <dependency>
            <groupId>mdht.hl7.ds4p</groupId>
            <artifactId>org.hl7.security.ds4p.contentprofile</artifactId>
            <version>${ds4pcontent.version}</version>
        </dependency>

ETT Validator's referenceccdavalidator/pom.xml has them: 
https://github.com/siteadmin/reference-ccda-validator/blob/master/pom.xml#L64

MDHT official releases have these 2 relatively new jars in the mdht folder, e.g.: MDHT_CCDA_R2.1_R1.1_MU2_DS4P_20180531_build\mdhtruntime\mdht

Your updated ais-validator also has these 2 jars, see 
https://github.cerner.com/ais/ais-validator/blob/master/dependencies.gradle#L80 

I understand our referenceccdavalidator is not using the features in these 2 jars yet.  Still, since we are updating the MDHT jars set, it makes sense to include all jars from MDHT that our reference ETT Validator (11/27/2018 ETT Validator release v1.0.37) has.  Thank you.

Reply:
added the 2 jars